test/util/entries.ts

```diff
@@ -1,16 +1,26 @@
 import type {Page} from '@playwright/test';
-import type {Metric} from '../../src';
+import type {Metric, CancellationError} from '../../src';
 
 // Use window.entries to communicate between browser and test processes
 declare global {
   interface Window {
     entries: Metric[];
+    errors: CancellationError[];
   }
 }
 
 /** Get the list of performance entries that have been recorded from the browser */
 export const getEntries = (page: Page) => page.evaluate(() => window.entries);
 
+/** Get the list of performance entries and errors that have been recorded from the browser */
+export const getEntriesAndErrors = (page: Page) =>
+  page.evaluate(() => {
+    return {
+      entries: window.entries,
+      errors: window.errors,
+    };
+  });
+
```

Comment:
Nit: Why not just make this `getErrors` and call both methods from a test if you want both lists?  This seems slightly simpler.

Reply:
I hear you! I was planning on a follow up PR that will change all instances of playwright testing to use `getEntriesAndErrors` and deprecate `getEntries` entirely - I think that will be more useful. I started doing that as part of this PR, but the diff would have been quite large.